aries/aries_vcx/src/common/primitives/credential_schema.rs

```diff
@@ -54,21 +54,14 @@ impl Schema {
             data
         );
 
-        let data_str = serde_json::to_string(data).map_err(|err| {
-            AriesVcxError::from_msg(
-                AriesVcxErrorKind::SerializationError,
-                format!("Failed to serialize schema attributes, err: {}", err),
-            )
-        })?;
-
         let schema_json = anoncreds
```

Comment:
nitpick: rename to `schema`

Reply:
All these `*_json` will be renamed in batch 👌